server/base/src/main/java/org/apache/accumulo/server/util/CleanZookeeper.java

```diff
@@ -67,25 +48,26 @@ public static void main(String[] args) {
             String instanceNamePath = root + Constants.ZINSTANCES + "/" + instanceName;
             byte[] id = zk.getData(instanceNamePath);
             if (id != null && !new String(id, UTF_8).equals(context.getInstanceID().canonical())) {
-              try {
-                zk.recursiveDelete(instanceNamePath, NodeMissingPolicy.SKIP);
-              } catch (KeeperException.NoAuthException ex) {
-                log.warn("Unable to delete {}", instanceNamePath);
-              }
+              delete(zk, instanceNamePath);
+              System.out.println("Deleted instance " + instanceName);
```

Comment:
```suggestion
              System.out.println("Deleted instance: " + instanceName);
```

Reply:
Thanks for the review, I should be able to push that quick change tomorrow when I get a chance.